src/actions/index.js

```diff
@@ -1,10 +1,7 @@
 // Copyright 2019 Stanford University see LICENSE for license
-export const setAppVersion = (version) => ({
-  type: "SET_APP_VERSION",
-  payload: version,
-})
-
-export const setCurrentComponent = (rootSubjectKey, rootPropertyKey, key) => ({
+const setCurrentComponent = (rootSubjectKey, rootPropertyKey, key) => ({
   type: "SET_CURRENT_COMPONENT",
   payload: { rootSubjectKey, rootPropertyKey, key },
 })
+
+export default setCurrentComponent
```

Comment:
Following our existing pattern, please add a `export const noop = () => {}` so that `setCurrentComponent` does not need to be default export.